docs/source/support/faq.rst

```diff
@@ -229,9 +229,9 @@ This is **ONLY** a heuristic to ensure that your node is running correctly, it d
     #. Run the following command and ensure that the various settings are correct::
 
         nucypher status stakers
-        >    --provider <your_geth_provider>
-        >    --network gemini
-        >    --staking-address <your_staker_address>
+        >    --provider <YOUR PROVIDER URI>
+        >    --network <NETWORK>
```

Comment:
:+1: - The generic sweet spot.

Reply:
Yep! We have a lot of other spots with `goerli`, `gemini`, `chainid 5` references 😱 .